airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorMetadataDefinitionV0.py

```diff
@@ -133,6 +140,13 @@ class BreakingChangeScope(BaseModel):
     )
 
 
+class RemotePackageIndex(BaseModel):
+    class Config:
+        extra = Extra.forbid
+
+    pypi: Optional[Pypi] = None
+
+
```

Comment:
Shall we go "polymorphic" here?
```suggestion

    index_name:  # Would be an enum like Pypi | Maven etc
    url: str # Would be the url to the package
```

```

Reply:
True, what do you think about using `RemoteRegistries` instead of `RemotePackageIndexes`:
```
remoteRegistries:
  pypi:
    url: <url to pypi package>
  DockerHub:
    url: <url to dockerHub images>
    imageAdress: dockerio.
  maven:
...
```

Dockerhub, maven etc. can come later of course.

Reply:
No strong opinion on the wording, changing to `remoteRegistries`